patch/kconfig-exclusions

```diff
@@ -19,6 +19,11 @@ CONFIG_THERMAL_STATISTICS
 CONFIG_UBIFS_FS_ZSTD
 CONFIG_CRYPTO_DEV_SAFEXCEL
 
+[pensando]
+CONFIG_ARM64_VA_BITS_48
+CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU
+CONFIG_SQUASHFS_FILE_DIRECT
```

Comment:
why exclude these options?

Reply:
Because these kernel config options are under "choice" section. For ex:
```
choice
	prompt "File decompression options"
	depends on [SQUASHFS]
	help
	  Squashfs now supports two options for decompressing file
	  data.  Traditionally Squashfs has decompressed into an
	  intermediate buffer and then memcopied it into the page cache.
	  Squashfs now supports the ability to decompress directly into
	  the page cache.

	  If unsure, select "Decompress file data into an intermediate buffer"

config [SQUASHFS_FILE_CACHE]
	bool "Decompress file data into an intermediate buffer"
	help
	  Decompress file data into an intermediate buffer and then
	  memcopy it into the page cache.

config [SQUASHFS_FILE_DIRECT]
	bool "Decompress files directly into the page cache"
	help
	  Directly decompress file data into the page cache.
	  Doing so can significantly improve performance because
	  it eliminates a memcpy and it also removes the lock contention
	  on the single buffer.

endchoice
```

In pensando case, CONFIG_SQUASHFS_FILE_CACHE is enabled. But by default in sonic kernel, SQUASHFS_FILE_DIRECT is enabled (which we don't want in pensando platform). This is similar w.r.t other options mentioned here

Reply:
why we cannot use FILE_DIRECT but not FILE_CACHE?

Reply:
Since pensando kernel was based on buildroot (https://github.com/buildroot/buildroot) its default in buildroot which was inherited and has been tested till now.

Reply:
let's align with generic sonic kernel options, let's not use CACHE.

Reply:
Tested the changes with FILE_DIRECT and it looks good. Updated commit